### PR TITLE
docs: Add description for Events and Highlights

### LINKED ISSUE #92: Content: Replace all dummy text fields in the given mock of events page
## Description
As a user,
I need brief yet engaging descriptions for all events displayed in cards and the heading EVENTS and HIGHLIGHTS,
so that I can walk through all of them and participate.

## Mocks
event-con-2

### Update [Required]
- [ ] The description below the main heading of this section should encourage the user to check out all our events and participate in upcoming or present ones
- [ ] Should have details of all the events conducted by the organization so far.

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
3-5 days

### PR DESCRIPTION
**Pull request by siddhipandare**
### Added description for events in the Events and Highlights Page 

Added content for Events and Highlights in src/events_and_highlights.json file. 
Added  4 supporting asset images in src/assets/events_and_highlights folder, namely 1.png, 2.png, 3.png and 4.png for the respective events :
* Intro to Open source
* Engaging community in Quality assurance
* Hopper India, 2020
* Grace Hopper Celebration
 
Fixes #92 

### Type of Change:

- Documentation

### How Has This Been Tested?
Checked if the JSON is valid or not using jsonlint.com 

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] Any dependent changes have been merged

### CODE REVIEW

**Comment by annabauza** [comment]
I don't understand this images.

**Comment by siddhipandare** [comment]
> @siddhipandare I don't understand this images.

Author said to add promotional images ([Ref](https://github.com/anitab-org/anitab-org.github.io/issues/92#issuecomment-660595105)) in the Event Highlights section.  
Why I Contribute to Open Source_LinkedIn (1)
They are promotional flyers like this one^^ 😓

File: src/content/events_and_highlights.json [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+              {
+                "par": "This event will give you an understanding of the motivation behind individuals who contribute to open source"
+              },
+              {
+                "par": "Open Source contributors with varying experiences will be there to share their journey and answer your questions!"
+              }
+            ]
+          },
+          {
+            "Event Highlights": "src\\assets\\events_and_highlights\\1.png"
```
Comment (annabauza) [inline src/content/events_and_highlights.json:44]: we will not be able to load it like that, please check how it was done in conent/*.js files with `require`. Thanks.

File: src/content/events_and_highlights.json [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+        },
+        {
+          "par": "Our events aim to give members insights from industry experts and female-identifying leaders on a variety of topics promoting Open Source, and covering stages of Open Source development right from Introduction to Quality Assurance."
+        }
+      ]
+    },
+    {
+      "title": "Events",
+      "content": [
+        [
```
Comment (annabauza) [inline src/content/events_and_highlights.json:17]: this structure looks incorrect:
should be:
```
event:{ title:"bla bla bla", date:"bla bla bla", etc..}
```

Comment (siddhipandare) [reply src/content/events_and_highlights.json:17]: Is this schema okay? I also corrected the relative path to the image string. I think this is how it is needed in the .json file.
😅 

```{
      "Events": [
        {
          "Event Title": "..",
          "Event Date": "..",
          "Event Location": "...",
          "Event Timings": "...",
          "Event description": [
            {
              "par": "...."
            }
          ],
         "Event Highlights": "../src/assets/events_and_highlights/1.png",
          "Know more": "....."
       } 
  ]
}
```

Comment (annabauza) [reply src/content/events_and_highlights.json:17]: In keys we do not use uppercase and we use spaces. if `title` is nested in `event` there is no need to repeat `event` in key.

**Comment by siddhipandare** [comment]
Hello, can anyone review this? I haven't squashed the commits because the last time I had tried I messed up the commit history.. can anyone please help me?

**Comment by annabauza** [comment]
@siddhipandare please update labels from Changes Requested to Needs Review. Thank you.

**Pull request merged**